scalikejdbc-core/src/main/scala/scalikejdbc/SQLInterpolationString.scala

```diff
@@ -14,7 +16,18 @@ class SQLInterpolationString(val s: StringContext) extends AnyVal {
     SQL[A](syntax.value).bind(syntax.parameters: _*)
   }
 
-  def sqls(params: Any*): SQLSyntax = SQLSyntax(buildQuery(params), buildParams(params))
+  def sqls(params: Any*): SQLSyntax = {
+    // Convert mutable collections to immutable Lists here to avoid
+    // mutation from another thread, which might cause a mismatch of
+    // the number of placeholders ("?") and parameters.
+    val convertedParams = params.map {
```

Comment:
The converted result doesn't have mutable 'collections', but might contain mutable objects. so `immutable` would be misleading. I feel `fixedParams` is fine. will rename it.